seismic/ASDFdatabase/FederatedASDFDataSetDBVariant.py

```diff
@@ -220,6 +220,21 @@ def decode_tag(tag, type='raw_recording'):
         # end for
     # end func
 
+    def get_global_time_range(self, network, station, location=None, channel=None):
+        query = "select min(st), max(et) from wdb where net='%s' and sta='%s' "%(network, station)
+
+        if (location):
+            query += "and loc='%s' "%(location)
+        if (channel):
+            query += "and cha='%s' "%(channel)
+
+        row = self.conn.execute(query).fetchall()[0]
+
+        min = UTCDateTime(row[0]) if row[0] else UTCDateTime(4102444800.0)
```

Comment:
Consider moving these min and max datetime constants to a file and importing them, since they appear more than once.

Reply:
Thanks; done.

Reply:
Merging now.